server/common/src/main/java/com/alipay/sofa/lookout/server/common/es/operation/ESOperator.java

```diff
@@ -157,6 +202,7 @@ void doRollOver(String alias) {
                     RequestBody.create(MediaType.parse("application/json"),
                         rollOverContent.getBytes("UTF-8"))).build();
             Response response = client.newCall(request).execute();
+            logger.info("rollOver:{}", request.toString());
```

Comment:
This operation is very frequent. I think it's better to change the  log level to debug.

Reply:
Change to bug level